.env.example

```diff
@@ -31,6 +31,7 @@ PUSH_NOTIFICATIONS_APN_ENV=development
 PUSH_NOTIFICATIONS_APN_TEAM_ID=
 PUSH_NOTIFICATIONS_APN_KEY_ID=
 PUSH_NOTIFICATIONS_APN_TOPIC=
+PUSH_NOTIFICATIONS_APN_KEY_PATH=/app/iosKey.p8

```

Comment:
Please document this new environment variable in `README.md` according to this format:

https://github.com/amida-tech/amida-notification-microservice#variable_name-required-if-it-actually-is-the-default-value